pkg/utils/resources.go

```diff
@@ -43,7 +42,7 @@ func AreResourcesDifferent(current, desired interface{}) bool {
 	desiredType := reflect.TypeOf(desired)
 
 	if currentType != desiredType {
-		logrus.Warnf("Attempting to compare resources for different types [%v] and [%v]", currentType, desiredType)
+		log.Info("Attempting to compare resources for different types ", "current", currentType, "desired", desiredType)
```

Comment:
if we are not supporting "warn" then this would be better to be changed to a debug

Reply:
`debug` also not supported at the moment

Reply:
is this an error? if so make it an error otherwise keep it at Info, but if you feel like it doesn't inform anything to a customer consider removing it.

Reply:
Looks like error for me, but not sure. Rewrite to the `error`

Reply:
Delete that message - if resources are not of the same type then clearly they are different. If the caller wanted to check for type consistency they would call a function that can return information about type consistency.

Reply:
Delete delete delete. The function is testing if 2 things are different. If they have different types then clearly they are different. If the caller needs info about type consistency they need to call a different function.